src/components/AppBar.scss

```diff
@@ -1,6 +1,7 @@
 .AppBar {
     & > div {
         display: inline-block;
+        z-index: 2;
```

Comment:
Nah, I think it's fine, just a pity that we need to hard code stuff like that because of the tools/frameworks/libraries/whatever.